tests/test_validate.py

```diff
@@ -1111,3 +1111,48 @@ def test_internal_component_metadata_copyright_festo_no_supplier(self) -> None:
             sbom["specVersion"] = spec_version
             issues = validate_test(sbom)
             self.assertEqual(search_for_word_issues("supplier", issues), True)
+
+
+class TestValidateFilename(unittest.TestCase):
```

Comment:
Is not there some redundancy to the tests in [`TestValidateInit`](https://github.com/Festo-se/cyclonedx-editor-validator/blob/3b6899609b48c3676ac92e9910e3e91d05471aba/tests/test_validate.py#L66)?

Reply:
You're right. I didn't think to look in there. Do you know why it's named _Init_?

I propose to delete the old tests. They hook into the validation command at a higher level, so they do test a couple of lines of code that aren't covered by the new tests but those should arguably be covered by integration tests in #157. On the other hand the new tests contain significantly more test cases.
Okay?

Reply:
As long as you mean to remove only the redundant tests, when yes, I'm fine with it. I would really like to keep the tests for the `warnings-ng` and `gitlab-cq-report`, as these are two essential functions for us, so we should at least know that they are working.